src/app/static/src/js/store/git/mutations.ts

```diff
@@ -9,6 +9,6 @@ export enum GitMutation {
 export const mutations: MutationTree<GitState> = {
     [GitMutation.SetMetadata](state: GitState, payload: RunReportMetadata) {
         state.metadata = payload.metadata;
-        state.git_branches = payload.git_branches;
```

Comment:
renamed this, I think we should keep all state props camelCased for consistency and maybe later down the line we look at serialising all api responses from orderly server so that the json arrives camelCased